src/backend/gl/src/conv.rs

```diff
@@ -159,6 +159,7 @@ pub fn describe_format(format: Format) -> Option<FormatDescription> {
             Integer,
         ),
         Rg16Sint => FormatDescription::new(glow::RG16I, glow::RG_INTEGER, glow::SHORT, 2, Integer),
+        Rg16Unorm => FormatDescription::new(glow::RG16, glow::RG_INTEGER, glow::UNSIGNED_SHORT, 2, Float),
```

Comment:
this should use `glow::RG` instead, it's not an integer format

Reply:
Hey thanks for the quick feeback, appreciate it!

I assume you're referring to the second parameter (`tex_external`), however no matter which of the two was meant, with either change I run into the second, later issue I described above. I've tried quite a few combinations and I'm unsure if it's really just this line that has an issue or if there's other code that needs adapting (there are only conversions for *8Unorms currently, I was wondering if that is maybe because for the other variants - 16/32 there would have to be more changes elsewhere?)

If your feeling is that this line in itself should be fine to enable usage or Rg16Unorm data, I'll hunt the other issue down the stack some more, because maybe in the end that other one is completely unrelated and I'm just wasting our time, which I really don't want to do. :) ... Thanks for your help!

Reply:
Are you referring to "Unsupported uniform datatype!" panic? How do we get there?
Do you have a repro case that I could try as well?

Reply:
Yes that one! I can look into the exact code path up to there a bit later (however as I said maybe this is really unrelated after all, might have been to quick in assuming a connection).

Re. repro: You can check out the `gl` branch of https://codeberg.org/suchbliss/good-form and run with `cargo run --features=gl`, which should start and then immediately crash with the mentioned error. (It's not exactly a minimal repro though I'm afraid ;))

Reply:
Oh re. repro l just realized with all the audio dependencies involved and whatnot ... I'm not sure the repro procedure on the whole project will be approachable depending on your system, I'll definitely look into it later anyhow!